### ISSUE TITLE
File browser or manager

### ISSUE THREAD

**Issue by dberenbaum** [iterative/studio-support#65]
@pmrowla and I discussed that it would be helpful to be able to see the structure of your data and repo inside Studio. This could look similar to `dvc ls` but in a nice UI. Some benefits:

* Enable anyone to easily get acclimated with a project
* Introducing DVC cloud storage without asking users to look through the raw cloud files
* Showing what's stored by DVC, including directories down to the the individual file level
* Combining Git-tracked files and DVC-tracked files in one view

**Comment by dberenbaum** [comment]
Some related features that would be nice to have:
* Get URL for any file.
* Download any data.
* Diff between commits (could be at the file level like `dvc diff`, at least as a starting point).

**Comment by daavoo** [comment]
Something like https://dagshub.com/shcheklein/example-get-started/src/main/data ?

**Comment by dberenbaum** [comment]
> * Get URL for any file.
> 
> * Download any data.
> 
> * Diff between commits (could be at the file level like `dvc diff`, at least as a starting point).

Rather than a separate interface, it would be nice to have these available inside the experiments table for each data column. The URL is already available, although it is not that helpful for a directory since it only links to the `.dir` file. Would be nice to see the files for that directory, compare those across commits, and download the whole directory as a zip archive or similar.

**Comment by shcheklein** [comment]
Thanks for the feature request, Dave. From my experience with VS Code DVC tracked tree - I think we need to clarify the user story here. It's not clear now what do we actually expect tbh (that's why it's hard even to tell - should it be within a table, or like GH/DagsHub interface, etc)

**Comment by dberenbaum** [comment]
Makes sense @shcheklein. A "file browser" might not have that much value, but I'll try to think about whether some of the features here would be valuable additions to teams' data versioning workflows.

**Comment by dberenbaum** [comment]
User request for this: https://discord.com/channels/485586884165107732/563406153334128681/1196427756184027166
